compatibility.bs

```diff
@@ -56,18 +32,21 @@ urlPrefix: https://drafts.csswg.org/css-values-3/
 
 <pre class=link-defaults>
 spec:css-animations-1; type:property; text:animation-delay
-spec:css-borders-4; type:property; text:box-shadow
-spec:css-borders-4; type:property; text:border-radius
 spec:css-borders-4; type:property; text:border-bottom-left-radius
 spec:css-borders-4; type:property; text:border-bottom-right-radius
+spec:css-borders-4; type:property; text:border-radius
 spec:css-borders-4; type:property; text:border-top-left-radius
 spec:css-borders-4; type:property; text:border-top-right-radius
+spec:css-borders-4; type:property; text:box-shadow
 spec:css-conditional-3; type:at-rule; text:@media
 spec:css-display-3; type:value; for:display; text:flex
 spec:css-flexbox-1; type:value; text:inline-flex
 spec:css-syntax-3; type:dfn; text:invalid
 spec:filter-effects-1; type:property; text:filter
+spec:html; type:element; text:body
+spec:infra; type:dfn; text:string
```

Comment:
The new ambiguous string definition is breaking almost all WHATWG specs (and probably others too), so we should probably work to get the source spec updated to stop exporting it instead.

Reply:
I've decided to accept this for now, but I agree that this is a bad state to be in, but mostly for specifications not maintained by the WHATWG as they are not good about addressing warnings and errors.